src/clr-angular/progress/spinner/spinner.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Directive } from '@angular/core';
+
+export const SPINNER_BASE_CLASS = 'spinner';
+
+@Directive({
+  selector: '[clrSpinner]',
```

Comment:
A possible solution @Shijir mentioned is instead of us adding a new public API we could have the directive match on the existing CSS selector. `selector: '.spinner'`. This would fix anyone already using `.spinner` in existing applications without having to introduce a new public API.